docs/installation/ssl.md

```diff
@@ -70,6 +70,38 @@ http {
 }
 ```
 
+Or an apache.conf file similar to:
+```virtual-site.conf
+<VirtualHost *:80>
+    ServerAdmin you@domain.com 
+    ServerName monica.yourdomain.com
+
+    RewriteEngine on
+    RewriteCond %{SERVER_NAME} =monica.yourdomain.com
+    # redirect all requests to port 80 to port 443 using 308 code
+    RewriteRule ^ https://%{SERVER_NAME}%{REQUEST_URI} [END,QSA,NE,R=308]
+ </VirtualHost>
+```
+
+```virtual-site-ssl.conf
+<IfModule mod_ssl.c>
+<VirtualHost *:443>
+    ServerAdmin you@domain.com
+    ServerName monica.yourdomain.com
+
+    ProxyPreserveHost On
+    ProxyRequests Off
+    ProxyPass / http://localhost:3001/
+    ProxyPassReverse / http://localhost:3001/
+    RequestHeader add X-Forwarded-Proto https
+
+    SSLCertificateFile /etc/letsencrypt/live/monica.yourdomain.com/fullchain.pem
+    SSLCertificateKeyFile /etc/letsencrypt/live/monica.yourdomain.com/privkey.pem
+    Include /etc/letsencrypt/options-ssl-apache.conf
```

Comment:
What should be this file?
I think it miss:
```
SSLEngine on
SSLProxyEngine on
SSLProxyVerify None
SSLProxyCheckPeerCN Off
SSLProxyCheckPeerName Off
```

Reply:
I guess I am assuming this is setup by letsencrypt automatically, or the user knows what they are doing.  My file has:

 SSLEngine on

 SSLProtocol             all -SSLv2 -SSLv3
 SSLCipherSuite          ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256- GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256:ECDHE-ECDSA-AES128 -SHA:ECDHE-RSA-AES256-SHA384:ECDHE-RSA-AES128-SHA:ECDHE-ECDSA-AES256-SHA384:ECDHE-ECDSA-AES256-SHA:ECDHE-RSA-AES256-SHA:DHE-RSA-AES128-SHA256:DHE-RSA-AES128-SH A:DHE-RSA-AES256-SHA256:DHE-RSA-AES256-SHA:ECDHE-ECDSA-DES-CBC3-SHA:ECDHE-RSA-DES-CBC3-SHA:EDH-RSA-DES-CBC3-SHA:AES128-GCM-SHA256:AES256-GCM-SHA384:AES128-SHA2 56:AES256-SHA256:AES128-SHA:AES256-SHA:DES-CBC3-SHA:!DSS
 SSLHonorCipherOrder     on
 SSLCompression          off

 SSLOptions +StrictRequire

 LogFormat "%h %l %u %t \"%r\" %>s %b \"%{Referer}i\" \"%{User-agent}i\"" vhost_combined
 LogFormat "%v %h %l %u %t \"%r\" %>s %b" vhost_common